docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,40 @@
+| [Kotlin for Apache Spark](https://github.com/JetBrains/kotlin-spark-api)     | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building | - Layer of compatibility between [Apache Spark](https://spark.apache.org/) and Kotlin <br/> - Apache Spark data transformation operations in Kotlin-idiomatic code <br/> - Simple usage of Kotlin features, such as data classes and lambda expressions, in curly braces or method reference                                                                                                                                                                                                                                                                                                                                                                    |
+| [Lets-Plot](https://github.com/JetBrains/lets-plot)                          | Data exploration and visualization                                                                                       | - Plotting statistical data written in Kotlin <br/> - Support for [Kotlin Notebook](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook), [Datalore](https://datalore.jetbrains.com/), and [Jupyter with Kotlin Kernel](https://github.com/Kotlin/kotlin-jupyter#readme) <br/> - Compatible with the JVM, JS, and Python <br/> - Embedding charts in [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/) applications                                                                                                                                                                                                           | 
+| [KMath](https://github.com/mipt-npm/kmath)                                   | - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building                         | - Modular library to work with mathematical abstractions in [Kotlin Multiplatform](https://www.jetbrains.com/kotlin-multiplatform/) (JVM, JS, Native, and Wasm) <br/> - APIs for algebraic structures, mathematical expressions, histograms, and streaming operations <br/> - Interchangeable wrappers over existing Java and Kotlin libraries, including [ND4J](https://github.com/eclipse/deeplearning4j/tree/master/nd4j), [Apache Commons Math](https://commons.apache.org/proper/commons-math/), [Multik](https://github.com/Kotlin/multik), and others <br/> - Inspired by Python's [NumPy](https://numpy.org/) but allowing superior architecture designs |
```

Comment:
Hmm I'm confused about the last point here. What superior architectural design is allowed? Could we name at least one example in a concise way?

Reply:
Is it clearer?